web/src/Game.js

```diff
@@ -1,8 +1,10 @@
 import React, { useState, useEffect, useContext } from 'react';
 import { EthersContext } from './EthersContext.js';
+import UsedChars from './UsedChars';
+import WinState from './WinState';
+import GuessCounter from './GuessCounter';
```

Comment:
@brianwchou did you forget to include these files?

Reply:
No i took out those components and made it their own file.